src/storage/quota-strategy/GlobalQuotaStrategy.ts

```diff
@@ -0,0 +1,85 @@
+import type { TransformOptions } from 'stream';
```

Comment:
We need to discuss folder structure and naming. I think this should be inside of `storage/quota`.